local_land_charges_api_stub/documentation/local-land-charges-api.yml

```diff
@@ -628,62 +664,82 @@ definitions:
     description: 'The name of the Minister of the Crown, government department, local authority or other person by whom the charge is brought into existence'
     type: string
     minLength: 1
-    example: 'Exeter City Council'
+    example: 'City of Exeter'
```

Comment:
This should be left as is, it was in correct format.

Reply:
Restored. No idea what happened